source/_integrations/default_config.markdown

```diff
@@ -32,6 +32,7 @@ This integration is a meta-component and configures a default set of integration
 - [Scene](/integrations/scene/) (`scene`)
 - [Scripts](/integrations/script/) (`script`)
 - [Simple Service Discovery Protocol (SSDP)](/integrations/ssdp/) (`ssdp`)
+- [Stream](/integrations/stream/) (`stream`)
```

Comment:
I failed to see the dependency. Is this a mistake or have I overlooked something?

Reply:
Only the ihc.markdown file should be part of this PR. I only changed that file so i don' t understand why there are other files here. I tried updating my "ihcconfigflow" branch from current, but that did not help. Can i just select "delete file" here in the PR?